source/tutorials/moving-pictures.rst

```diff
@@ -327,6 +327,33 @@ Finally, ordination is a popular approach for exploring microbial community comp
 .. question::
     What differences do you observe between the unweighted UniFrac and Bray-Curtis PCoA plots?
 
+Alpha rarefaction plotting
+--------------------------
+
+In this section we'll explore alpha diversity as a function of sampling depth using the ``qiime diversity alpha-rarefaction`` visualizer. This visualizer computes one or more alpha diversity metrics at multiple sampling depths, in steps between 1 (optionally controlled with ``--p-min-depth``) and the value provided as ``--p-max-depth``. At each sampling depth step, 10 (optionally controlled with ``--p-iterations``) rarefied tables will be generated, and the diversity metrics will be computed for all samples in the tables. Average diversity values will be plotted for each sample at each even sampling depth, and samples can be grouped based on metadata in the resulting visualization if sample metadata is provided.
```

Comment:
> At each sampling depth step, 10 (optionally controlled with ``--p-iterations``) rarefied tables will be generated

The parenthesis near the start of the clause is a bit awkward in this sentence — maybe move to the end of the end of the sentence to improve readability?

> if sample metadata is provided

maybe add "(with the XXXX parameter as shown below)"